These changes keep track of the per-process RFS information.
***************
*** 140,145
  		unsigned pr_off;	/* pc offset */
  		unsigned pr_scale;	/* pc scaling */
  	} u_prof;
  	int	u_stack[1];
  };
  

--- 140,155 -----
  		unsigned pr_off;	/* pc offset */
  		unsigned pr_scale;	/* pc scaling */
  	} u_prof;
+ #ifdef REMOTEFS
+ 	short	u_rmtoffset[2];		/* path offset for local segment */
+ 	short	u_rmtcdir;		/* system # of remote current dir */
+ 	long	u_rmtsys;		/* room for 32 systems */
+ 					/* one bit for each remote system */
+ #ifdef pyr /* Pyramid */
+ 	int	*u.u_ap;
+ #define		u_arg	u_ap
+ #endif pyr
+ #endif REMOTEFS
  	int	u_stack[1];
  };
  
